src/app/shared/services/share/share.service.ts

```diff
@@ -7,13 +7,20 @@ import { FileManagerService } from "../file-manager/file-manager.service";
 import { TemplateAssetService } from "../../components/template/services/template-asset.service";
 import { Capacitor } from "@capacitor/core";
 
-const SHARE_NOT_SUPPORTED_ON_PLATFORM_ERROR_MESSAGE =
-  "[SHARE] Sharing is not supported on this platform";
+interface IShareActionParams {
+  file?: string;
+  text?: string;
+  url?: string;
+  title?: string;
+  dialog_title?: string;
+}
 
 @Injectable({
   providedIn: "root",
 })
 export class ShareService extends SyncServiceBase {
+  // Temporary local storage path on native devices for file being shared
+  localFilepath: string;
```

Comment:
nit(non-blocking): if this is not required by external components it should be marked as private. Also better to use `/** */` style comments for intellisense